bcgov/mds
# Feature/mds 4029 - Project Summary Notifications

# Main

- Added email to ministry staff after project summary is created in MineSpace

# Other

- N/A

# How to test
- Log into MineSpace as a BCeID user and submit a project summary, ensure that your local email client is enabled.


# Notes
![Screen Shot 2021-12-06 at 3 50 51 PM](https://user-images.githubusercontent.com/25125247/144940861-bed96fd2-fca8-4d0d-a03c-3765e11fa6f0.png)
![Screen Shot 2021-12-06 at 3 50 15 PM](https://user-images.githubusercontent.com/25125247/144940819-9627184e-0e4d-4f82-bfd2-ecb06e0f878f.png)
